README.Rmd

```diff
@@ -32,7 +32,7 @@ an unreliable source means a date should be flagged as uncertain,
 or sources offer multiple, competing dates.
 
 As researchers, we often recognise this messiness
-but are forced to force nonexistent precision on data to we can proceed with analysis.
+but are forced to force nonexistent precision on data so we can proceed with analysis.
```

Comment:
Should the cheatsheet go anywhere in readme? Do we have a good to go cheatsheet? Last I saw it need only a few small changes, maybe we can wait for @EstherPeev to be back on Monday so that she can update that and we can add the cheatsheet to the readme before merging the PR, what do you think?